### ISSUE TITLE
Android: Unable to launch the application after the latest update

### ISSUE THREAD

**Issue by katsumoto84** [koush/vysor.io#1261]
After the last update, every time I try to launch the application I get the error: The procedure entry point DiscardVirtualMemory could not be located in the dynamic link library KERNEL32.dll

I'm on windows 7

If I install an older version it works until it automatically updates, after which the problem starts again. I basically have to reinstall the program every time I turn my computer back on

**Comment by drivetheory** [comment]
same problem here on my Vysor Pro license using Win7.

**Comment by koush** [comment]
Windows 7 is no longer supported by even Microsoft. Sorry. I gotta keep up to date with security requirements and that means new electron builds that may not work on old OS.

**Comment by katsumoto84** [comment]
I paid for a product and you can't force me to change system to continue using it… You must at least give the possibility to continue using an old version with automatic updating disabled

**Comment by katsumoto84** [comment]
> With all due respect that is a COMPLETELY BULLSHIT answer. you force product updates. For those of us who paid for the software you should provide the ability to disable those updates and continue to run the software we purchased that was compatible for the operating system we bought it for......

exactly,You have the duty to guarantee me the possibility of using an old version, if there are any security risks, that's my business, I freely choose to use it despite the risks

**Comment by koush** [comment]
set the installation directory to read only or something
